### ISSUE TITLE
Display consistent error message when invalid catalogId is specified.

### ISSUE THREAD

**Issue by jonathancrabtree** [nih-cfde/dashboard#39]
Currently the pages, for the most part, fail silently when passed an invalid catalog id. The dcc_review.html page displays a simple JavaScript alert message when pointed at a catalog with multiple DCCs' data.

**Comment by jrchudy** [comment]
In the default case, no catalogId is supplied, this appears to work without error now. When you pass `catalogId=1` the same page is seen as if there's no catalogId. When looking at a specific catalog ([catalog 208 on production](https://app.nih-cfde.org/dcc_review.html?catalogId=208)), there's no error. The only error case now is when an invalid catalogId is given. I don't think this is an issue anymore.

@jonathancrabtree If that makes sense to you, then we can close this issue.

**Comment by jrchudy** [comment]
https://app.nih-cfde.org/dcc_review.html?catalogId=216

When displaying multiple DCCs we don't error anymore but there are still issues with the dcc_review page with showing content related to only the first one. Is the data relevant to both DCCs or just the first one?

**Comment by mschor** [comment]
Hi @jrchudy and @ACharbonneau, this ticket is assigned to me, but I don't feel like I have clarity on what is needed here. Are we changing the DCC Review page to be handle more than one DCC somehow and if so, what does that look like? Or, are we redirecting them somewhere else and if so, where?

**Comment by jrchudy** [comment]
We discussed this more in depth today in the UX meeting. The 2 likely scenarios that would cause an error on the dcc_review page are:
 - the user being logged out then clicking on a link that brings them to the dcc_review page
 - the catalog the dcc_review page is being loaded with no longer exists

Any other failure modes shouldn't occur from links generated by the application, so we shouldn't worry too much about those. For instance, a user could view the review page for the released data by putting the catalog id into the url themself.

Unless future requirements change, the dcc page is only going to be used to show the information for 1 DCC. Closing this issue.
